src/components/UI/WorkspaceSwitcher.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <select v-model="selectedWorkspace" @change="this.updateActiveWorkspace">
+      <option value="">Personal Workspace</option>
+      <option v-for="workspace in this.workspaces" :key="workspace.id" :value="workspace.shortcode">{{workspace.name}}</option>
+    </select>
+    <user-properties ref="userProperties"></user-properties>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapState } from "vuex";
+
+export default {
+  name: "WorkspaceSwitcher",
+  computed: {
+    ...mapState('auth', ["activeWorkspace", "user"]),
```

Comment:
`user` is not being used anywhere.

Reply:
It is, right below in the workspaces computed property

Reply:
Sorry my bad!